backend/src/app.py

```diff
@@ -1,15 +1,19 @@
 from flask import Flask, make_response
```

Comment:
On autoformatting:
* I can't remember the exact details we discussed, but what I see is that auto-formatting works in JS but not in Python, which I _think_ matches what you see
* First I checked that I do have a Python formatter installed, I actively added the Black-formatter extension from Microsoft but there was a default one already
* But looking at the Black formatter output (in the "output" tab of the terminal)
> 2024-09-13 11:09:15.445 [info] The language client requires VS Code version ^1.82.0 but received version 1.79.2

Updating my VSCode (and it was about time I did!) fixed the issue, so maybe you're seeing the same?

Reply:
In fact it's actually the JS side that's not working for me 😁 . I'll have a bit more of a look into it. Aware that trying to get it in in this PR might cause a lot of unnecessary formatting changes to clutter up the actual changes.